src/pages/mesh/advanced/index.md

```diff
@@ -389,7 +382,6 @@ Org ID: 123456789
 Project ID: 1234567890123456789
 Workspace ID: 2345678901234567890
 Mesh ID: 12a3b4c5-6d78-4012-3456-7e890fa1bcde
-Legacy Mesh Endpoint: https://graph.adobe.io/api/123456-123-456-789-1234567890/graphql?api_key=09876543210987654321
 Edge Mesh Endpoint: https://edge-graph.adobe.io/api/123456-123-456-789-1234567890/graphql
```

Comment:
We're removing the word `Edge` from all labels. Essentially "Legacy Mesh" goes away and "Edge Mesh" becomes "Mesh". 
```suggestion
Mesh Endpoint: https://edge-graph.adobe.io/api/123456-123-456-789-1234567890/graphql
```